locations/spiders/five_below.py

```diff
@@ -0,0 +1,86 @@
+import scrapy
+import re
+
+from locations.hours import OpeningHours
+from locations.items import GeojsonPointItem
+
+class FiveBelowSpider(scrapy.Spider):
+    name = "five_below"
+    allowed_domains = [ "locations.fivebelow.com" ]
+    start_urls = (
+        "https://locations.fivebelow.com/sitemap.xml",
+    )
+
+    def start_requests(self):

```

Comment:
The default behavior is to send requests for the `start_urls` list. You don't need to define `start_requests()` if you're not overriding the default.

Reply:
Thanks.  I submitted an update that removes `start_requests()`